### DISCUSSION TITLE
Adding variant for <select> with blank option selected

### DISCUSSION THREAD

**Discussion by swanson** [tailwindlabs/tailwindcss#9554] [Help]
https://play.tailwindcss.com/WGb0iiaLdI?file=config

Trying to add a variant for `select-empty` -- where you have a `<select>` tag but the option currently selected has a value of `""`. I think I'm missing something basic but cannot figure out the right magic for the variant definition

**Comment by swanson** [comment]
Maybe this is actually not possible with CSS until subject selectors are supported.

**Reply by adamwathan** [reply]
I think you can probably do this with `:has` but tough to check from my phone. No Firefox support for that yet though sadly.

**Reply by adamwathan** [reply]
So this kind of works but the styles apply forever, not just when the empty option is selected. Tried adjusting it to use `[selected]` as well but the browser doesn't seem to handle it the way I'd expect, the styles apply forever even after selecting a different option.

https://play.tailwindcss.com/YkEdwgGanU?file=config
